plutus-core/executables/src/PlutusCore/Executable/Common.hs

```diff
@@ -430,18 +429,18 @@ loadUplcASTfromFlat flatMode inp = do
 getProgram ::
     ( ProgramLike p
     , Functor p
-    , PLC.Rename (p PLC.SourcePos)
+    , PLC.Rename (p PLC.SrcSpan)
     ) =>
     Format ->
     Input ->
-    IO (p PLC.SourcePos)
+    IO (p PLC.SrcSpan)
 getProgram fmt inp =
     case fmt of
         Textual -> parseInput inp
         Flat flatMode -> do
             prog <- loadASTfromFlat flatMode inp
             -- No source locations in Flat, so we have to make them up.
-            return $ PLC.topSourcePos <$ prog
+            return $ PLC.SrcSpan "top" 1 1 1 2 <$ prog
```

Comment:
I would prefer if we kept this as a constant named topSrcSpan like we did before with topSourcePos